deployment/provision_vagrant_vm.sh

```diff
@@ -46,6 +46,7 @@ sudo -H -u $USER $ENV_FOLDER/bin/pip install mod_wsgi
 
 # setup apache
 a2enmod expires
+a2enmod headers
```

Comment:
https://stackoverflow.com/questions/5799906/what-s-the-difference-between-expires-and-cache-control-headers sounds like Cache-Control more or less superseded Expires. maybe we can remove the latter?

Reply:
Only disadvantage I could imagine would be that the numbers are not that readable as they were before. Sample configuration (not tested nor commited yet):
```
    <Directory ${REPO_FOLDER}/evap/static_collected>
        Header set Cache-Control "max-age: 10800" // 3 hours

        <FilesMatch "\.[0-9a-f]{12}\.\w+$">
            Header set Cache-Control "max-age: 31536000, immutable" // 1 year
        </FilesMatch>
    </Directory>
```

Reply:
looks good to me!

Reply:
Despides some modifications to the syntax (comments need an own line and `max-age` wants a `=` not a `:` after it), I tested this locally and seams to work